events/tests.py

```diff
@@ -91,3 +93,36 @@ def test_event_save(self):
             },
         )
         self.assertEqual(Event.objects.last().name, "I am a test event")
+
+
+class TestEventListView(TestCase, Mixin):
+    def setUp(self):
+        today = now()
+        event_place = self.create_place(
+            venue="lollipop", address="Eeifel tower", is_online=False
+        )
+        category = self.create_category(category_name="funny")
+        user = self.create_user()
+
+        self.event = self.create_event(
+            description="Testing is good",
+            place=event_place,
+            image="Users/admin/Desktop/Screenshot 2021-09-09 at 5.10.43 PM.png",
+            category=category,
+            user=user,
+            event_name="test event",
+        )
```

Comment:
This code is not different from 
```
Event.objects.create(
            name=event_name,
            description=description,
            image=image,
            place=place,
            category=category,
            user=user,
  )
```
Except the first line, remaining all are same. Main reason I asked to create a method in mixin is to remove redundant parameters.